spec/docs/npq_participants_spec.rb

```diff
@@ -54,4 +55,81 @@
       end
     end
   end
+
+  path "/api/v1/participants/npq/{id}/withdraw" do
+    put "Withdrawn a participant from a course" do
+      operationId :npq_participants
+      tags "NPQ participant"
+      security [bearerAuth: []]
+      consumes "application/json"
+
+      request_body content: {
+        "application/json": {
+          "schema": {
+            "$ref": "#/components/schemas/NPQParticipantWithdrawalRequest",
```

Comment:
This has a wrong name (`NPQParticipantWithdrawRequest`) seems to be what it is called everywhere else :)